client-unity/Assets/Scenes/Main.unity

```diff
@@ -191,7 +191,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
```

Comment:
By default now the elements underneath the username chooser are disabled by default and then are only enabled if the user has to created a new player (technically they have to choose a username).